### ISSUE TITLE
[Instagram] yt-dlp trying to load available video formats for both videos and photos in a carousel instead of only for videos, newly includes photos in playlist indexes

### ISSUE THREAD

**Issue by ianlpearson** [yt-dlp/yt-dlp#7569]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

USA

### Provide a description that is worded well enough to be understood

Example URL: https://www.instagram.com/p/CtXtwOop1W5/

Issue seems to apply to Instagram carousels that are a mix of photos and videos. If the post is only one video with no photos, or the carousel is all videos with no photos, all is well.

What's the issue:

Previously, yt-dlp attempted to load formats only for the actual videos. Now, it includes photos in the 'downloading X of 10' index count. For example, the second video in a carousel that previously had index of 2 may now have index of 5 if there are 3 photos before it. This is an issue for us because in our use case we assume the playlist-index options will only include videos, as it did previously. We're putting a workaround in place (simply include photos in our playlist-items counter we supply to yt-dlp) but thought you guys should know!

In the example logs, if I recall correctly, previously it would have only attempted to download the two successful videos, as index 1 and 2 and with no other entries. Instead we see the videos at indexes 6 and 10 with 'no video formats found' errors for the photos that make up the rest of the carousel.

It still downloads the videos at least if playlist index is not supplied, and has no issues if the new-style indexes including the photos are supplied (example included under the ======= in the logs)

Issue is happening on previous versions (march 2023), current release (2023.07.06) and latest master at this time (  git commit 2af4eeb77246b8183aae75a0a8d19f18c08115b2 ). Logs are from 2023.07.06.

Issue is happening both under Linux x86 and Mac arm64 environments. Haven't tried Windows. Happens under several different IG accounts from several different IP addresses, too.

Ideal fix: return to only including actual videos in the list of items, so that we return to 1,2 instead of 6,10 as indexes in the example post, and no longer throw an error about video formats not being available for photos.

I tried to take a look myself but I'm definitely in way over my head. Thank you all for any insight you can provide!!

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
./yt-dlp --cookies cookies.txt https://www.instagram.com/p/CtXtwOop1W5/ -vU --ignore-config 
[debug] Command-line config: ['--cookies', 'cookies.txt', 'https://www.instagram.com/p/CtXtwOop1W5/', '-vU', '--ignore-config']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (zip)
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.3 (CPython arm64 64bit) - macOS-13.4.1-arm64-arm-64bit (OpenSSL 1.1.1t  7 Feb 2023)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1856 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 639e1fdead08745ef6c11d704358c5f4ba37e198b960fddf29cf38fae63a3807
yt-dlp is up to date (stable@2023.07.06)
[Instagram] Extracting URL: https://www.instagram.com/p/CtXtwOop1W5/
[Instagram] CtXtwOop1W5: Downloading video info
[download] Downloading playlist: Post by johnnygreenoge
[Instagram] Playlist Post by johnnygreenoge: Downloading 10 items of 10
[download] Downloading item 1 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 2 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 3 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 4 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 5 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 6 of 10
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] CtXs3CepBOR: Downloading 1 format(s): 1232091700785727v
[debug] Invoking http downloader on "https://scontent-den4-1.cdninstagram.com/v/t50.2886-16/353554460_783870060144535_2825624437081297308_n.mp4?efg=eyJ2ZW5jb2RlX3RhZyI6InZ0c192b2RfdXJsZ2VuLjcyMC5jYXJvdXNlbF9pdGVtLmJhc2VsaW5lIiwicWVfZ3JvdXBzIjoiW1wiaWdfd2ViX2RlbGl2ZXJ5X3Z0c19vdGZcIl0ifQ&_nc_ht=scontent-den4-1.cdninstagram.com&_nc_cat=107&_nc_ohc=smF95r8lKXgAX_TYR1-&edm=ALQROFkBAAAA&vs=1232091700785727_3133586636&_nc_vs=HBksFQAYJEdCelFFaFdYSjN6NDdNZ0NBSnpSNUdoMG9qWW5ia1lMQUFBRhUAAsgBABUAGCRHUFJVdWhTVk52OHdlSGNYQURWbzZZMjA5MEVyYmtZTEFBQUYVAgLIAQAoABgAGwGIB3VzZV9vaWwBMBUAACbmvdX7g4fxPxUCKAJDMywXQDddsi0OVgQYEmRhc2hfYmFzZWxpbmVfMV92MREAde4HAA%3D%3D&_nc_rid=bb5d8243a4&ccb=7-5&oh=00_AfDyCtWhYKRnJ-CWn68mA4TUxlJuM4F_H9NK-OiR8B6v8Q&oe=64AF799E&_nc_sid=fc8dfb"
[download] Video by johnnygreenoge [CtXs3CepBOR].mp4 has already been downloaded
[download] 100% of    3.78MiB
[download] Downloading item 7 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 8 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 9 of 10
ERROR: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2014, in __process_iterable_entry
    return self.process_ie_result(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/Users/ianpearson/dev/yt-dlp/./yt-dlp/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [Instagram] CtXtwOop1W5: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U

[download] Downloading item 10 of 10
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] CtXs3CnplI6: Downloading 1 format(s): 281775690900352v
[debug] Invoking http downloader on "https://scontent-den4-1.cdninstagram.com/v/t50.2886-16/353369701_1720713331718145_525540286358984753_n.mp4?efg=eyJ2ZW5jb2RlX3RhZyI6InZ0c192b2RfdXJsZ2VuLjcyMC5jYXJvdXNlbF9pdGVtLmJhc2VsaW5lIiwicWVfZ3JvdXBzIjoiW1wiaWdfd2ViX2RlbGl2ZXJ5X3Z0c19vdGZcIl0ifQ&_nc_ht=scontent-den4-1.cdninstagram.com&_nc_cat=109&_nc_ohc=J7BnITs-ARQAX-fQMIT&edm=ALQROFkBAAAA&vs=281775690900352_3093801407&_nc_vs=HBksFQAYJEdHWF9EeFVCdU9YS19od0dBREdZbFNNYkdFc0hia1lMQUFBRhUAAsgBABUAGCRHS3I5RmhXellUVXJxY0FFQUxTSHppZjRUczktYmtZTEFBQUYVAgLIAQAoABgAGwGIB3VzZV9vaWwBMBUAACa%2B6%2B2r6YStQBUCKAJDMywXQE4AAAAAAAAYEmRhc2hfYmFzZWxpbmVfMV92MREAde4HAA%3D%3D&_nc_rid=bb5d8f7064&ccb=7-5&oh=00_AfCr2EufStpxOr_9mvvPzwhFpYyyWP0OVicNShYS-Qc-Aw&oe=64AF24E2&_nc_sid=fc8dfb"
[download] Video by johnnygreenoge [CtXs3CnplI6].mp4 has already been downloaded
[download] 100% of    6.75MiB
[download] Finished downloading playlist: Post by johnnygreenoge



=====================================================================================

There are no issues when the changed playlist-items index is supplied (e.g. now 6,10 instead of 1,2)

./yt-dlp --cookies cookies.txt https://www.instagram.com/p/CtXtwOop1W5/ -vU --ignore-config --playlist-items 6,10
[debug] Command-line config: ['--cookies', 'cookies.txt', 'https://www.instagram.com/p/CtXtwOop1W5/', '-vU', '--ignore-config', '--playlist-items', '6,10']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (zip)
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.3 (CPython arm64 64bit) - macOS-13.4.1-arm64-arm-64bit (OpenSSL 1.1.1t  7 Feb 2023)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0
[debug] Optional libraries: sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1856 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 639e1fdead08745ef6c11d704358c5f4ba37e198b960fddf29cf38fae63a3807
yt-dlp is up to date (stable@2023.07.06)
[Instagram] Extracting URL: https://www.instagram.com/p/CtXtwOop1W5/
[Instagram] CtXtwOop1W5: Downloading video info
[download] Downloading playlist: Post by johnnygreenoge
[Instagram] Playlist Post by johnnygreenoge: Downloading 2 items of 10
[download] Downloading item 1 of 2
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] CtXs3CepBOR: Downloading 1 format(s): 1232091700785727v
[debug] Invoking http downloader on "https://scontent-den4-1.cdninstagram.com/v/t50.2886-16/353554460_783870060144535_2825624437081297308_n.mp4?efg=eyJ2ZW5jb2RlX3RhZyI6InZ0c192b2RfdXJsZ2VuLjcyMC5jYXJvdXNlbF9pdGVtLmJhc2VsaW5lIiwicWVfZ3JvdXBzIjoiW1wiaWdfd2ViX2RlbGl2ZXJ5X3Z0c19vdGZcIl0ifQ&_nc_ht=scontent-den4-1.cdninstagram.com&_nc_cat=107&_nc_ohc=smF95r8lKXgAX_TYR1-&edm=ALQROFkBAAAA&vs=1232091700785727_3133586636&_nc_vs=HBksFQAYJEdCelFFaFdYSjN6NDdNZ0NBSnpSNUdoMG9qWW5ia1lMQUFBRhUAAsgBABUAGCRHUFJVdWhTVk52OHdlSGNYQURWbzZZMjA5MEVyYmtZTEFBQUYVAgLIAQAoABgAGwGIB3VzZV9vaWwBMBUAACbmvdX7g4fxPxUCKAJDMywXQDddsi0OVgQYEmRhc2hfYmFzZWxpbmVfMV92MREAde4HAA%3D%3D&_nc_rid=a4da59d25a&ccb=7-5&oh=00_AfDyCtWhYKRnJ-CWn68mA4TUxlJuM4F_H9NK-OiR8B6v8Q&oe=64AF799E&_nc_sid=fc8dfb"
[download] Video by johnnygreenoge [CtXs3CepBOR].mp4 has already been downloaded
[download] 100% of    3.78MiB
[download] Downloading item 2 of 2
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] CtXs3CnplI6: Downloading 1 format(s): 281775690900352v
[debug] Invoking http downloader on "https://scontent-den4-1.cdninstagram.com/v/t50.2886-16/353369701_1720713331718145_525540286358984753_n.mp4?efg=eyJ2ZW5jb2RlX3RhZyI6InZ0c192b2RfdXJsZ2VuLjcyMC5jYXJvdXNlbF9pdGVtLmJhc2VsaW5lIiwicWVfZ3JvdXBzIjoiW1wiaWdfd2ViX2RlbGl2ZXJ5X3Z0c19vdGZcIl0ifQ&_nc_ht=scontent-den4-1.cdninstagram.com&_nc_cat=109&_nc_ohc=J7BnITs-ARQAX-fQMIT&edm=ALQROFkBAAAA&vs=281775690900352_3093801407&_nc_vs=HBksFQAYJEdHWF9EeFVCdU9YS19od0dBREdZbFNNYkdFc0hia1lMQUFBRhUAAsgBABUAGCRHS3I5RmhXellUVXJxY0FFQUxTSHppZjRUczktYmtZTEFBQUYVAgLIAQAoABgAGwGIB3VzZV9vaWwBMBUAACa%2B6%2B2r6YStQBUCKAJDMywXQE4AAAAAAAAYEmRhc2hfYmFzZWxpbmVfMV92MREAde4HAA%3D%3D&_nc_rid=a4da5ec771&ccb=7-5&oh=00_AfCr2EufStpxOr_9mvvPzwhFpYyyWP0OVicNShYS-Qc-Aw&oe=64AF24E2&_nc_sid=fc8dfb"
[download] Video by johnnygreenoge [CtXs3CnplI6].mp4 has already been downloaded
[download] 100% of    6.75MiB
[download] Finished downloading playlist: Post by johnnygreenoge
```

**Comment by bashonly** [comment]
Note that this only happens when logged in; this is playlist still has 2 entries when not passing cookies

**Comment by bashonly** [comment]
Closing as `wontfix`; this is just how it will be going forward.

Use `--ignore-no-formats-error` if you don't like the errors. You can use that together with `--write-thumb` to download the images from the non-video carousel items (and all image-only posts).

> Note that this only happens when logged in

This will no longer be true after #17075 is merged; the API response we will get when logged-out is much more aligned with the logged-in API response now.
